src/jogos/jogos.repository.ts

```diff
@@ -13,4 +13,10 @@ export class JogosRepository {
 
         return await jogoCriado.save();
     }
+
+    async findAll() {
+        const jogos = this.jogoModel.find()
```

Comment:
O await deve ser feito aqui, pois é onde o Mongo está retornando uma promise.

        const jogos = await this.jogoModel.find()